client/Routes.js

```diff
@@ -26,6 +24,7 @@ class Routes extends Component {
         {isLoggedIn ? (
           <Switch>
             <Route path="/home" component={Home} />
+            <Route path="/dashboard" component={Home} />
```

Comment:
is line 26 removed?

Reply:
Hmmm, it isn't removed on my end. I think we should keep both since the component names are Home page?